### DISCUSSION TITLE
Possible to invalidate/provide tags between reducers?

### DISCUSSION THREAD

**Discussion by mweiss9676** [reduxjs/redux-toolkit#1384] [Q&A]
I have my application split into multiple reducers with their own reducerPaths. When I provide tags in one reducerPath and invalidate them in another nothing happens. For instance I have an 'Orders' tag being provided in my 'api/admin/orderitems' reducerPath and I'm trying to invalidate that tag in my 'api/admin/orders' reducerPath. Is this possible to do?

**Comment by phryneas** [comment]
It is not intended. If they should invalidate each other, they clearly belong into the same api. Generally, you should have only one api except in very rare edge cases where you talk to different servers with completely different datasets.

**Reply by mweiss9676** [reply]
Really appreciate the fast response! They all do share the same base URL in this case so I was able to refactor them by using the injectEndpoints api. I guess I could see a use case for reaching across APIs though, there would be no workaround for that I take it.

**Reply by phryneas** [reply]
If you reach "across APIs", from an application logic standpoint they are the same api, as they share an internal dataset. There might be rare cases where you just have endpoint descriptions with absolute urls and cannot take advantage of a baseUrl, but that would not be then end of the world ;)

It would be much worse when unrelated apis would suddenly start invaldiating each other ^^

**Reply by mweiss9676** [reply]
I see your point but I would say it's extremely easy to come up with a unique tag in the case that you didn't want to share those tags but their is no workaround if you want to share them in the current implementation. I agree, it doesn't seem like that big of a deal right now, but one minor inconvenience I've noticed since refactoring is that I could no longer use my dotnet core inspired simple API names such as "get" and "delete" because they were no longer unique being under the same umbrella now. so now they have to be "getOrderItems" which no longer has a direct corelated endpoint API side in the app. Again, minor stuff here but it seems like it's more limiting this way and more verbose and the downside of sharing tags would be very simple to avoid by just using a more unique tag. 

Just thoughts here, btw and for the record I love this API, really reducers boilerplate and the automated fetching and statuses are a big help in avoiding annoying issues I've dealt w/ in the past. Great work!
